__tests__/app.test.js

```diff
@@ -387,88 +387,126 @@ describe("get /api/reviews?category=<categoryName>", () => {
-      expect(returnedObj).toHaveProperty("created_at",expect.any(String));
-    })
-  })
-
-  test("Returns a 201 status, and the uploaded object has the correct specific values where possible to check (Will also have a review_id, commend_id, votes, and created_at)", ()=>{
-    const inputObj = {username: "mallionaire", body: "This was decent, not the best not the worst"};
-    
-    return request(app)
-    .post("/api/reviews/10/comments")
-    .send(inputObj)
-    .expect(201).then((res)=>{
-      
-      let returnedObj = res.body.comment;
-      expect(returnedObj.author).toBe(inputObj.username);
-      expect(returnedObj.body).toBe(inputObj.body);
-      expect(returnedObj.votes).toBe(0);
-    })
-  })
-
-  test("Returns a 400 status, when endpoint is not formatted correctly", ()=>{
-    const inputObj = {username: "mallionaire", body: "This was decent, not the best not the worst"};
-    
-    return request(app)
-    .post("/api/reviews/a/comments")
-    .send(inputObj)
-    .expect(400)
-  })
-
-  test("Returns a 400 status, when given an non-existent key", ()=>{
-    const inputObj = {name: "mallionaire", body: "This was decent, not the best not the worst"};
-    
-    return request(app)
-    .post("/api/reviews/1/comments")
-    .send(inputObj)
-    .expect(404)
-  })
-
-  test("Returns a 404 status, when given an invalid type of value on a valid key", ()=>{
-    const inputObj = {username: 2, body: "This was decent, not the best not the worst"};
-    
-    return request(app)
-    .post("/api/reviews/1/comments")
-    .send(inputObj)
-    //Should this be 404 or 400?
-    .expect(404)
-  })
-
-  test("Returns a 404 status, if given a username that is not in the foreign key", ()=>{
-    const inputObj = {username: "josh", body: "This was decent, not the best not the worst"};
-    
-    return request(app)
-    .post("/api/reviews/1/comments")
-    .send(inputObj)
-    .expect(404)
-  })
-
-  test("Returns a 400 status, if not given an object to attach", ()=>{
-    return request(app)
-    .post("/api/reviews/1/comments")
-    .expect(404)
-  })
+  test("returns a 201 status", () => {
+    const inputObj = {
+      username: "mallionaire",
+      body: "This was decent, not the best not the worst",
+    };
+
+    return request(app)
+      .post("/api/reviews/10/comments")
+      .send(inputObj)
+      .expect(201);
+  });
+  test("returns a 201 status, with an attached object (with correct keys", () => {
+    const inputObj = {
+      username: "mallionaire",
+      body: "This was decent, not the best not the worst",
+    };
+
+    return request(app)
+      .post("/api/reviews/10/comments")
+      .send(inputObj)
+      .expect(201)
+      .then((res) => {
+        let returnedObj = res.body.comment;
+
+        expect(returnedObj).toHaveProperty("comment_id", expect.any(Number));
+        expect(returnedObj).toHaveProperty("body", expect.any(String));
+        expect(returnedObj).toHaveProperty("review_id", expect.any(Number));
```

Comment:
would expect the review_id to be 10, not any. Otherwise this would pass for the wrong object being returned